docs/src/pages/demos/dividers/InsetDividers.tsx

```diff
@@ -11,19 +10,19 @@ import WorkIcon from '@material-ui/icons/Work';
 import BeachAccessIcon from '@material-ui/icons/BeachAccess';
 import Divider from '@material-ui/core/Divider';
 
-const styles = (theme: Theme) =>
+const useStyles = makeStyles((theme: Theme) =>
   createStyles({
```

Comment:
We should no longer need `createStyles`

Reply:
This part is blur for me. I thought it was #15402 removing this `createStyles` requirement?

Reply:
That looks to only change the resolve path for makeStyles? I don't think that would affect `createStyles`. https://github.com/mui-org/material-ui/pull/15366#pullrequestreview-227250770 looks more likely?

Reply:
@oliviertassinari Downside of me making it in different PRs, I pointed that out here: https://github.com/mui-org/material-ui/pull/15486#discussion_r278677267

Reply:
I confirm that the autocompletion works without:

![Capture d’écran 2019-04-25 à 22 04 15](https://user-images.githubusercontent.com/3165635/56764894-271db080-67a6-11e9-9524-25915aa965cd.png)

Reply:
The issue is in `makeStyles`, `withStyles` is working fine for me as well.

> We will need to cleanup the usage of the method form all our demos.

Will do as soon as https://github.com/mui-org/material-ui/pull/15402 is merged